README.md

```diff
@@ -172,7 +172,7 @@ To enable per component debugging set the "debug" localstorage key with one or m
 - `cms` this will print info to the console as changes are made to authored content via the CMS
 - `document` this will add the active document as `window.currentDocument`, you can use MST's hidden toJSON() like `currentDocument.toJSON()` to views its content.
 - `drop` console log the dataTransfer object from drop events on the document.
-- `history` this will print some info the console as the history system records changes in the document.
+- `history` this will: print some info the console as the history system records changes, prints the full history as JSON each time it is loaded from Firestore, and provide a `window.historyDocument` so you can inspect the document while navigating the history.
```

Comment:
```suggestion
- `history` this will: print some info to the console as the history system records changes, print the full history as JSON each time it is loaded from Firestore, and provide a `window.historyDocument` so you can inspect the document while navigating the history.
```